.babelrc

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
- [ ] You are not using [Gitflow](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/git-github/articles/gitflow.md) 😨 Gitflow means you make your **dev** branch default branch and whenever you wanna make any change to your application you make a **feature** branch from your **dev** branch and after completing your work and getting an approval you merge that **feature** branch to your **dev** branch. But here you are using Github Flow 👍 Kindly use the correct flow 😊